plugins/modules/aci_rest.py

```diff
@@ -61,6 +61,17 @@
     - Preserve the response for the provided path.
     type: bool
     default: false
+  page_size:
```

Comment:
shoudl we add example including with pagination included?

Reply:
Added an example with 5 tenants pagination